client/src/components/TeamsAddDialog.vue

```diff
@@ -103,7 +117,15 @@ export default {
     },
     isCoop() {
       return this.game ? this.game.coop || this.coop : null
-    }
+    },
+    filteredTeams(){
```

Comment:
could you move it to composition-api as well? There is a [ticket](https://github.com/DavidGolodetsky/GeekScore/issues/135) about rewriting the whole component to composition-api. You can take it if you want as you are already aware with its logic

Reply:
ah, I see why you did it, cause you rely on `this` keyword. Okay then this is out of the scope of this ticket, I can merge it now